Makefile

```diff
@@ -9,7 +9,7 @@ TFGEN := pulumi-tfgen-$(PACK)
 PROVIDER := pulumi-resource-$(PACK)
 VERSION := $(shell pulumictl get version)
 JAVA_GEN := pulumi-java-gen
-JAVA_GEN_VERSION := v0.5.4
+JAVA_GEN_VERSION := v0.9.5
```

Comment:
this needs a ci-mgmt followup - should this be either centrally managed, or a config option?

Reply:
This should be a config option in repo, but that's larger question. I'll update ci-mgmt after I get the lint to pass.